linux_os/guide/system/apparmor/grub2_enable_apparmor/oval/shared.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="grub2_enable_apparmor" version="1">
+    {{{ oval_metadata("Ensure AppArmor is enabled in the bootloader configuration") }}}
```

Comment:
Just working through this and I was wondering if the [`grub2-bootloader-argument` template](https://complianceascode.readthedocs.io/en/latest/templates/template_reference.html#grub2-bootloader-argument) might help in this case.

Reply:
hmm, is there a way to pass two arguments and values to this template?

Reply:
I don't think there is. In that case, you should have two rules.

Reply:
then I believe it is better to keep it as-is without templates